src/api/web-api.types.ts

```diff
@@ -27,3 +27,22 @@ export interface User {
   email: string;
   roles: UserRole[];
 }
+
+export type UserAction = 'LOGIN' | 'LOGOUT' | 'CHANGE_PASSWORD' | 'UPDATED';
```

Comment:
would be nice to use Prisma types, but when I looked into it, it was pretty awkward and harder to setup than expected.